be/src/storage/vectorized/tablet_reader.cpp

```diff
@@ -33,14 +33,13 @@ void TabletReader::close() {
 
 Status TabletReader::prepare() {
     _tablet->obtain_header_rdlock();
-    auto res = _tablet->capture_consistent_rowsets(_version, &_rowsets);
+    auto st = _tablet->capture_consistent_rowsets(_version, &_rowsets);
     _tablet->release_header_lock();
-    if (res == OLAP_SUCCESS) {
-        return Status::OK();
-    } else {
-        return Status::InternalError(
+    if (!st.ok()) {
+        st = Status::InternalError(
```

Comment:
1. Why not return st directly?
2. If return other status, it's better to print st.message()

Reply:
updated